### PR TITLE
Add test coverage for pending EP

### PR DESCRIPTION
**Pull request by allthesignals**
Adds a test for a scenario in which there are pending EPs. Ensures it offramps and skips MAS calls.

Also, in the submit_form526 spec, restricts multiple EVSS calls for pending EPs.

## Description of change
Increases test coverage

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

File: spec/jobs/rapid_ready_for_decision/form526_base_job_spec.rb [hunk 1]
```diff
@@ -116,6 +116,17 @@
             end
           end
         end
+
+        it 'skips forwarding the claim to MAS' do
+          VCR.use_cassette('evss/claims/claims') do
+            Sidekiq::Testing.inline! do
+              expect(MailAutomation::Client).not_to receive(:new)
```
Comment (yoomlam) [inline spec/jobs/rapid_ready_for_decision/form526_base_job_spec.rb:123]: To DRY the code, consider moving this line to the prior test and renaming the prior test `'off-ramps to the non-RRD process and does not notify MAS'`, and removing this test.

**Review by yoomlam — CHANGES_REQUESTED** [review]
1 suggestion

**Review by LindseySaari — APPROVED** [review]

**Comment by yoomlam** [comment]
@nanotone Do we want this PR or can it be closed?

**Comment by nanotone** [comment]
@yoomlam Yeah I'm not convinced more integration tests is the best path forward, at this point. I'm going to close it.

**Pull request closed without merging**